.github/workflows/push.yml

```diff
@@ -88,7 +88,7 @@ jobs:
                     - keboola.dbt-transformation-remote-mssql
                     - keboola.dbt-transformation-remote-bigquery
                     - keboola.dbt-transformation-remote-redshift
-                dbt_version: [ 1.7.1, 1.6.8, 1.5.9, 1.4.9 ]
+                dbt_version: [ 1.8.6, 1.7.1, 1.6.8, 1.5.9, 1.4.9 ]
                 # MSSQL is not supported yet in dbt 1.7.1 and 1.6.8
                 exclude:
```

Comment:
Jestli to funguje na 1.8.6, tak by mělo i na těch starších ne? Tak zkus ten exclude vyhodit.

Reply:
Zkoušel jsem to ale nefungovalo, https://github.com/keboola/dbt-transformation/actions/runs/11012844022, takže jsem to tam zase vrátil

Reply:
Ok, tak to je fakt divný, že u toho latest to najednou jde 🤷‍♂️ :D